assignments/objects.js

```diff
@@ -17,48 +17,126 @@ const example = {
   "gender": "F"
 }
 
+console.log("Objects Begin")
+console.log("challenge 1");
 // Write your intern objects here:
+const mitzi = {
+  "id": 0,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const kennan = {
+  "id": 1,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+}
+
+const keven = {
+  "id": 2,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
 
+const gannie = {
+  "id": 3,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+
+const antonietta = {
+  "id": 4,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+}
 
+console.log(mitzi, kennan, keven, gannie, antonietta);
+
+console.log();
+
+console.log("challenge 2");
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(mitzi.name);
 // Kennan's ID
-
+console.log(kennan.id);
 // Keven's email
-
+console.log(keven.email);
 // Gannie's name
-
+console.log(gannie.name);
 // Antonietta's Gender
+console.log(antonietta.gender);
+
+console.log();
 
+console.log("challenge 3");
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
 
+kennan.speak = function() {
+  return `Hello, my name is ${this.name}!`;
+}
+
+console.log(kennan.speak());
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
-//console.log(antonietta.multiplyNums(3,4));
 
+antonietta.multiplyNums = function(num1, num2) {
+  return num1 * num2;
+}
+
+console.log(antonietta.multiplyNums(3,4));
 // === Great work! === Head over to the the arrays.js file or take a look at the stretch challenge
 
+console.log();
+console.log("stretch begins");
+
 // ==== Stretch Challenge: Nested Objects and the this keyword ==== 
 
 // 1. Create a parent object with properties for name and age.  Make the name Susan and the age 70.
 // 2. Nest a child object in the parent object with name and age as well.  The name will be George and the age will be 50.
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  name: "Susan",
+  age: 70,
+
+  child: {
+    name: "George",
+    age: 50,
+
+    grandchild: {
+      name: "Sam",
+      age: 30
+    }
+  }
+}
+
+parent.speak = parent.child.speak = parent.child.grandchild.speak = function() {
```

Comment:
`=` is one of the operators that have right to left associativity in JavaScript. What are the others? Well there is the ternary operator `condition ? true_clause : false_clause` which has a right to left associativity, which allows building of complex conditional expressions (not statements but expressions).